ui/DesignSystem/Component/Notification.svelte

```diff
@@ -1,67 +1,72 @@
 <script>
-  import { Icon, Title } from "../Primitive";
+  import { Icon, Text, Title } from "../Primitive";
 
-  export let icon = Icon.Info;
   export let style = null;
-
   export let level = "info";
+  export let noIcon = false;
 </script>
 
 <style>
   .notification {
     display: flex;
-    flex: 1;
-    border-radius: 2px;
-    height: 40px;
+    border-radius: 4px;
+    height: 32px;
     align-items: center;
   }
 
   .info {
-    border: 1px solid var(--color-positive);
-    color: var(--color-positive);
-    background-color: var(--color-positive-level-1);
+    color: var(--color-background);
+    background-color: var(--color-foreground);
   }
 
   .notification.info :global(svg) {
-    fill: var(--color-positive);
-  }
-
-  .warning {
-    border: 1px solid var(--color-caution);
-    color: var(--color-caution);
-    background-color: var(--color-caution-level-1);
-  }
-
-  .notification.warning :global(svg) {
-    fill: var(--color-caution);
+    fill: var(--color-background);
   }
 
   .error {
-    border: 1px solid var(--color-negative-level-6);
-    color: var(--color-negative-level-6);
-    background-color: var(--color-negative-level-1);
+    color: var(--color-background);
+    background-color: var(--color-negative);
   }
 
   .notification.error :global(svg) {
-    fill: var(--color-negative-level-6);
+    fill: var(--color-background);
   }
 
   .close {
```

Comment:
If you chose to make the "action" configurable, I'd rename all the `close` classes to `action`.

Reply:
For a different time ;)